src/blocks/Alert.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import CommonMessage, { CommonMessageProps } from "./shared/CommonMessage"
+
+import "./Alert.scss"
+
+interface AlertProps extends Omit<CommonMessageProps, "role" | "closable"> {}
+
+const Alert = (props: AlertProps) => {
+  const classNames = ["alert"]
+  if (props.className) classNames.push(props.className)
+
+  return <CommonMessage {...props} role="alert" closable className={classNames.join(" ")} />
```

Comment:
Curious if there ever is a case where we wouldn't want an Alert to be "closable". Or is it best practice to always have an alert be closable?

Reply:
I'm guessing a non-closable "alert" would be less, er, alert-y, in which case it's probably better to use Message. But we can double-check with @slowbot.